Makefile

```
@@ -0,0 +1,21 @@
+.PHONY: down deploy shell up
+
+#setup image tag to latest for dev
+IMAGE_TAG?=latest
+DEPLOY_TYPE?=dev
+
+# stop containers
+down:
+	sudo docker compose down
+
+# run the deploy script, passing in deployment type
+deploy:
+	sudo ./deploy.sh -d $(DEPLOY_TYPE)
+
+# exec into api container
+shell:
+	sudo docker compose exec api sh
```

```suggestion
	docker compose exec api sh
```

Do we need sudo here? Isn't it better to run it as normal user and ensure the user has been included in the docker group or something?